src/functionalTest/resources/updatecase/valid-request-with-auto-case-update-fields.json

```diff
@@ -0,0 +1,120 @@
+{
+  "is_automated_process": false,
+  "exception_record": {
```

Comment:
Best to get rid of `exception_record`, as it's being replaced by `case_update_details`

Reply:
The spec says 
`exception_record section will be removed once all services have switched to using case_update_details section. In the meantime, both sections will be present in the request.`